WordPress/Classes/Services/PlanService.swift

```diff
@@ -28,3 +28,28 @@ extension PlanService {
         self.init(remote: remote)
     }
 }
+
+typealias PlanFeatures = [PlanID: [PlanFeature]]
+
+struct PlanFeaturesService {
+    private static var planFeatures = PlanFeatures()
+    
+    /// @returns All features that are part of the specified plan
+    static func featuresForPlan(plan: Plan) -> [PlanFeature] {
+        return planFeatures[plan.id] ?? []
+    }
+
+    /// @returns The feature that is part of the specified plan, with a matching slug (if one exists).
+    static func featureForPlan(plan: Plan, withSlug slug: String) -> PlanFeature? {
+        return featuresForPlan(plan).filter({ $0.slug == slug }).first
+    }
+    
+    private let remote = PlanFeaturesRemote(api: WordPressComApi.anonymousApi())
```

Comment:
I don't think it makes much of a difference now, but I think we should use a remote with the current account's api.